Geometry/TrackerCommonData/plugins/dd4hep/DDCutTubsFromPoints.cc

```diff
@@ -106,8 +107,8 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
       float P3_z_l = (P1_z_l + P2_z_l) / 2;
       float P3_z_t = (P1_z_t + P2_z_t) / 2;
 
+      std::string segname(solidOutput + "_seg_" + std::to_string(segment));
       edm::LogVerbatim("TrackerGeom").log([&](auto& log) {
-        std::string segname(solidOutput + "_seg_" + std::to_string(segment));
```

Comment:
In the code before this PR, the `segname` variable is defined and used in a debug message to describe the solids being created, but then it is NOT actually used. That would appear to me to be an oversight. Comparing to the old DD code, it seems clear this name is intended to be used as the name for the Cut Tubes. See below.